Mathlib/Algebra/Lie/Weights/Chain.lean

```diff
@@ -129,6 +175,43 @@ lemma trace_toEndomorphism_weightSpaceChain_eq_zero [IsNoetherian R L] [H.IsCart
     simp [hfg]
   · rw [LieModuleHom.map_add, LieHom.map_add, map_add, h₁, h₂, zero_add]
 
-end IsNilpotentSubalgebra
+/-- Given a (potential) root `α` relative to a Cartan subalgebra `H`, if we restrict to the ideal
+`I = (rootSpaceProductNegSelf α).range` of `H`, we may find an integral linear combination between
```

Comment:
Can we somehow remind the user what `rootSpaceProductNegSelf α).range` is? Maybe add `(= $[H_\alpha, H_{-\alpha}]$)` or something like that?

Reply:
Thanks, I agree; done.

Reply:
I'm even thinking that in future work I should perhaps drop the definition `rootSpaceProductNegSelf` in favour of a better-named definition for what is currently called `(rootSpaceProductNegSelf α).range` since this ideal is really the important thing.

I plan to think about this once I finish in this corner and can assess the general impact.